Miniscript-Template-000-Timelocks in Templates.mediawiki

```diff
@@ -0,0 +1,81 @@
+<pre>
+  Miniscript Template: 0
+  Title: Proposed Timelock Usage
+  Created: 2023-09-11
+</pre>
+
+
+1. Name of Template
+
+Proposed Timelock Usage
+
+2. Goal to be achieved by template
+
+Since [https://bitcoin.sipa.be/miniscript/ Miniscript] facilitates easier access to utilizing timelocks in Bitcoin, it opens a new frontier in Bitcoin security concerning the construction of more advanced Scripts. This proposal serves as an initial attempt to outline best practices for using timelocks in conjunction with Miniscript. The objective is to aid in wallet recovery and to standardize timelock use across various software and hardware wallet implementations.
+
+Should an output descriptor be lost or only partially recovered for any reason, minimizing the overall search space for expected timestamp values can expedite recovery. Although Miniscript supports any valid block height or epoch timestamp for absolute timelocks, the purpose of these proposed templates is to constrain the inherent open-endedness of Miniscript. This aims to foster user-friendly best practices in software wallets and signing devices.
+
+=Block height based Timelocks:=
+
+==Absolute block height based Timelocks==
+
+Proposal: Absolute block height based timelocks should be a multiple of 100, always ending in `00`.
+
+Examples of valid Block Height Absolute Timelocks
+
+*after(1000)
+*after(50700)
+*after(82800)
+*after(615000)
+
+
+== Relative block height based Timelocks ==
+Proposal: Relative block height timelocks should also be multiple of 100, with an exception for the maximum possible value supported, `65,536` (2^16) blocks.
+
+
+*older(100)
+*older(1500)
+*older(65536)
+
+
+=Epoch timestamp based Timelocks:=
+
+==Absolute Epochtime based Timelocks==
+
+*Since the goal of an epoch timestamp timelock is to align events with wall time instead of block time, epoch timestamps used should be of signifigance. Since there is a much larger searchspace with epoch timestamp timelocks, I'm proposing Epoch Timestamp based Timelocks should be set to either Noon, or Midnight GMT. 
+*In practice, this means that absolute epoch timelocks should be divisible by either `43,200` (Noon GMT) or `86,400` (Midnight GMT).
```

Comment:
In 4b75129bd282aa8fc81fe4dbd86c28f870b5ad8b:

You could go further and suggest midnight on Thursday (multiple of 604800) as an even-more preferable multiple. Though what's written is fine.